org.mixedrealitytoolkit.spatialmanipulation/ObjectManipulator/ObjectManipulator.cs

```diff
@@ -586,6 +603,14 @@ protected override void Awake()
             InstantiateManipulationLogic();
         }
 
+        /// <summary>
+        /// A Unity event function that is called on the frame when a script is enabled just before any of the update methods are called the first time.
+        /// </summary> 
+        private void Start()
```

Comment:
Another coding guideline we hadn't set in stone, but one that can cause an easy pitfall for derived classes: the recommendation that Unity messages should be `protected` 
https://github.com/microsoft/Microsoft.Unity.Analyzers/blob/main/doc/UNT0021.md

We didn't do a great job at this in MRTK2, but it had been a point of emphasis while we were developing MRTK3 with an eye for extensibility.

Some additional context at https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11643#discussion_r1240435925